api/jobset/v1alpha2/jobset_webhook.go

```diff
@@ -29,7 +29,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
 	"sigs.k8s.io/jobset/pkg/util/collections"
-	shared "sigs.k8s.io/jobset/pkg/util/shared"
+	shared "sigs.k8s.io/jobset/pkg/util/placement"
```

Comment:
I thought you had this change merged already, why is it showing up here?

Reply:
Needed to rebase, sorry about that. PR is updated now.